docs/content/concepts/video.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Video
+order: 2000
+---
+
+## Video in Rerun
+A stream of images (like those produced by a camera) can be logged to Rerun in several different ways:
+
+* Uncompressed, as many [`Image`](../reference/types/archetypes/image.md)s
+* Compressed as many [`EncodedImage`](../reference/types/archetypes/encoded_image.md)s, using e.g. JPEG-encoding
+* Compressed as a single [`AssetVideo`](../reference/types/archetypes/asset_video.md), using e.g. MP4 and AV1 or H.264.
```

Comment:
not sure if codecs should be listed here - that's easy to outdate and also not complete already